app/models/profile.rb

```diff
@@ -18,7 +18,7 @@ class Profile < ApplicationRecord
   enum deactivation_reason: {
     password_reset: 1,
     encryption_error: 2,
-    gpo_verification_pending: 3,
+    # gpo_verification_pending used to be here. Do not map anything to 3 for deactivation_reason
```

Comment:
one idea would be to rename it, instead of commenting out?

```suggestion
    gpo_verification_pending__NO_LONGER_USED: 3, # deprecated
```